ntoskrnl/include/internal/mm.h

```diff
@@ -46,6 +46,8 @@ extern SIZE_T MmtotalCommitLimitMaximum;
 extern PVOID MiDebugMapping; // internal
 extern PMMPTE MmDebugPte; // internal
 
+extern KSPIN_LOCK MmPfnLock;
```

Comment:
Is this for the annotations? IIRC those don't care about what you put there. You can probably use _Requires_lock_not_held_(LockQueuePfnLock)

Reply:
No, it's for the assert below.